### DISCUSSION TITLE
Suppressing False Positives

### DISCUSSION THREAD

**Discussion by RobSHK** [DependencyTrack/dependency-track#2982] [Q&A]
With one of the latest Dependency Check releases the "[Suppressing False Positives](https://jeremylong.github.io/DependencyCheck/general/suppression.html)" was introduced.
I wondered if the same could or already implemented with Dependency Track?

**Comment by officerNordberg** [comment]
Suppressing False positives has been in Dependency Check for years, so I'm not sure what may or may not be new about the latest release.  I know this discussion has been had here before.  I think @nscuro  may have a library to assist with this.  I was quickly able to find this https://pypi.org/project/dependency-track/

What we do is I wrote a webhook that uses documents similar to the following to apply False Positives to the Audit Trail.  I'm not saying there couldn't be better alternatives but this was pretty easy to implement and has the same "feel" as the odc suppression files.

```
    {
        "vulnIds": ["CVE-2021-25220", "CVE-2022-0396", "CVE-2019-6470"],
        "component_classifier": "LIBRARY",
        "payload":  {
            "analysisState": "NOT_AFFECTED",
            "analysisJustification": "CODE_NOT_PRESENT",
            "analysisDetails": "this CVE applies to bind server, we run bind client",
            "suppressed": true
        }
    },
    {
        "vulnIds": ["CVE-2010-3438", "CVE-2019-10794", "CVE-2022-24709"],
        "component_name": "component",
        "payload":  {
            "analysisState": "FALSE_POSITIVE",
            "analysisDetails": "bad fuzzy match",
            "suppressed": true
        }
    },
    
```

**Reply by RobSHK** [reply]
@officerNordberg thanks for your reply.
I'm affraid I was not clear enogh with my question.
What I meant was reffering to https://jeremylong.github.io/DependencyCheck/data/index.html
"Hosted base suppressions file
For a faster roundtrip time ([issue #4723](https://github.com/jeremylong/DependencyCheck/issues/4723)) to get false-positive report solution out to the users dependency-check starting from version 8.0.0 is using an online hosted [suppressions file](https://jeremylong.github.io/DependencyCheck/suppressions/publishedSuppressions.xml). For environments with constraints to internet access this file can be locally mirrored by customizing the hostedsuppressions file URL. See the tool-specific configuration documentation on the [github pages](https://jeremylong.github.io/DependencyCheck/index.html) for the exact advanced configuration flag to specify the custom location. Failure to download the hosted suppressions file will result in only a warning from the tool, but may result in false positives being reported by your scan that have already been mitigated by the hosted suppressions file."

Is there a way to include a ref. within Dependency Track to this online hosted file?
